File: back_up/arrays/arrays.go
package main

import (
	"fmt"
	"time"
)

func main() {
	sum := 0
	start := time.Now()

	for e := 0; e < 30; e++ {
		var x [1000000]int
		for i := 0; i < 1000000; i++ {
			x[i] = i
		}

		var y [1000000 - 1]int
		for i := 0; i < 1000000-1; i++ {
			y[i] = x[i] + x[i+1]
		}

		sum = 0
		for i := 0; i < 1000000; i += 100 {
			sum += y[i]
		}
	}

	elapsed := time.Since(start)

	fmt.Printf("Elapsed %s\n", elapsed)
	fmt.Println(sum)
}

File: back_up/arrays/arrays.py
import time

sum = 0
start = time.time()
for e in range(30):
    sum = 0

    x = [0] * 1000000
    for i in range(1000000):
        #x.append(i)
        x[i] = i

    y = [0] * 1000000
    for i in range(1000000 - 1):
        #y.append(x[i] + x[i+1])
        y[i] = x[i] + x[i+1]

    for i in range(0, 1000000, 100):
        sum += y[i]

end = time.time()
print('Elapsed ' + str(end-start))
print(sum)
